scripts/test.py

```diff
@@ -41,18 +41,26 @@ def monte_carlo_sampling_chaospy(N_FEATURES, SAMPLES, rule="latin_hypercube", se
     """
     import chaospy
     from scipy.stats import qmc 
+    from sklearn.preprocessing import MinMaxScaler
+
 
     # Generate a Nfeatures-dimensional latin hypercube varying between 0 and 1:
-    N_FEATURES = "chaospy.Uniform(0, 1), "*N_FEATURES
+    N_FEATURES = f"chaospy.{DISTRIBUTION}(0, 1), "*N_FEATURES
     uniform_cube = eval(f"chaospy.J({N_FEATURES})")  # writes Nfeatures times the chaospy.uniform... command)
     lh = uniform_cube.sample(SAMPLES, rule=rule, seed=seed).T
```

Comment:
@GbotemiB I assume, `uniform_cube` refers to the uniform distribution, right? If that is the case, could it probably make sense to adjust the variable name to avoid confusion?